ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
```

Comment:
Consider using a relative unit for width instead of a fixed pixel value for better responsiveness.